### PR TITLE
Reviewed the documentation

### PR DESCRIPTION
**Pull request by EstherChristopher**
1. Edited the topic of the documentation to Pascal Case.
2. Completed a sentence for clarity

### CODE REVIEW

**Comment by minrk** [comment]
Thanks for your contribution! This file now has conflicts because of an earlier contribution to the same file. If you resolve the conflicts, we can review these contributions.

**Comment by EstherChristopher** [comment]
Hello, thank you for reaching out. I've tried to resolve the conflict and update the PR by keeping only my branch changes. You can review it now and let me know what you think.

File: docs/source/events/index.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
-Event logging and Telemetry
-===========================
+patch-1
```
Comment (minrk) [inline docs/source/events/index.rst:1]: Looks like the conflict hasn't quite been fixed as there are remnants of hte branch names.

The heading also needs to keep the `====` underline.

**Review by minrk — COMMENTED** [review]
Thanks for updating! It looks like there are some remnants of the merge conflict left in the file.

We don't use "Title Case" in our headings, we try to use "Sentence case," which means there still is a change to fix here: lowercase `telementry`.

**Comment by EstherChristopher** [comment]
I'm not able to make the corrections you highlighted because I don't have access to the file. I tried clicking on the **View changes** link to see if I can get access to it and implement the corrections like I did the other time but it does not seem to be able to work. Can you find a way to give me the **Resolve conflicts** icon so I can adjust it like I previously did. This [link](https://github.com/jupyterhub/jupyterhub/pull/4109/files/e228b017e9d975abd8864058b13e5fd85ab2a0bb#diff-7c8f80b518905b7782a3f92d6ae0d580d79a813b294d3c07e420a8bc428e058f) also does not give me access to edit the file.

**Comment by minrk** [comment]
There are not conflicts, because the conflicts were resolved. Howver, they were resolved in such a way that they included some remnants of the conflict markers.

You can edit the file with [this link](https://github.com/jupyterhub/jupyterhub/pull/4109/files/e228b017e9d975abd8864058b13e5fd85ab2a0bb#diff-7c8f80b518905b7782a3f92d6ae0d580d79a813b294d3c07e420a8bc428e058f):
Screen Shot 2022-10-21 at 14 57 01

**Comment by EstherChristopher** [comment]
Ok, I understand, thank you. I have adjusted the title to sentence case.

**Comment by EstherChristopher** [comment]
Hi @minrk 
Please would the pull request be merged anytime soon. So I'd be able to make my final request.

**Comment by EstherChristopher** [comment]
Hi, @minrk Please can you point out exactly on what lines the texts of the conflict resolution are so I can remove them.

**Comment by minrk** [comment]
You should be able to see them in https://github.com/jupyterhub/jupyterhub/pull/4109/files . There are lines that look like 'main' or 'patch' which shouldn't be there. There should only be changes you intend highlighted in green.

**Comment by minrk** [comment]
Great, thank you!

**Pull request merged**